komodo/fetch.py

```diff
@@ -70,8 +70,13 @@ def fetch(pkgs, repo, outdir, pip="pip") -> dict:
 
     if not outdir:
         raise ValueError(
-            "The value of `outdir`, the cache location for pip and other "
-            "tools, cannot be None or the empty string."
+            "The value of `outdir`, the download destination location "
+            "cannot be None or the empty string."
+        )
+    if os.path.exists(outdir) and os.listdir(outdir):
+        print(
+            f"WARNING: Downloading to non-empty directory {outdir}, "
+            "this is not recommended."
```

Comment:
Actually, I think it's currently not _possible_, is it? I think the build will fail.

Reply:
At least it will fail if you use the protocol "nfs". Probably not for all other protocols, but behaviour is undefined. We should probably fail hard, but then we are at a breaking change.